mteb/evaluation/evaluators/Image/Any2TextMultipleChoiceEvaluator.py

```diff
@@ -7,15 +7,13 @@
 import torch
 from sklearn.metrics import accuracy_score
 from sklearn.metrics.pairwise import cosine_similarity
-from torchvision import transforms
 from tqdm import tqdm
 
 from mteb.encoder_interface import Encoder, EncoderWithSimilarity
 from mteb.evaluation.evaluators.Evaluator import Evaluator
 
 logger = logging.getLogger(__name__)
 
-transform = transforms.Compose([transforms.PILToTensor()])
```

Comment:
This does not seem to be used in the code

Reply:
same for the `self.transform`

Reply:
Let's remove it as well then eh?